.github/workflows/cancel.yml

```diff
@@ -30,7 +30,12 @@ jobs:
     name: "Cancel duplicate workflow runs"
     runs-on: ubuntu-latest
     steps:
-      - uses: apache/airflow-cancel-workflow-runs@953e057dc81d3458935a18d1184c386b0f6b5738
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
```

Comment:
Why do we need to checkout before cancelling?

Reply:
As per [this post](https://github.community/t/path-to-action-in-the-same-repository-as-workflow/16952), we need to get the code for the action (thus checking out with submodules) so that it can be run.